proto/operator.proto

```diff
@@ -106,29 +106,53 @@ message Notification {
   }
 }
 
-enum NotificationType {
-  NOTIFICATION_TYPE_UNSPECIFIED = 0;
-  NOTIFICATION_TYPE_CHANNEL_OPENING = 1;
-  NOTIFICATION_TYPE_CHANNEL_OPENED = 2;
-  NOTIFICATION_TYPE_CHANNEL_CLOSED = 3;
-  NOTIFICATION_TYPE_CHALLENGE_STARTED = 4;
-  NOTIFICATION_TYPE_CHALLENGE_CLEARED = 5;
-  NOTIFICATION_TYPE_CHALLENGE_FINISHED = 6;
-  NOTIFICATION_TYPE_MARGIN_UPDATED = 7;
-}
-
-enum ChannelStatus {
-  CHANNEL_STATUS_UNSPECIFIED = 0;
-  CHANNEL_STATUS_OPENING = 1;
-  CHANNEL_STATUS_OPEN = 2;
-  CHANNEL_STATUS_CHALLENGING = 3;
-  CHANNEL_STATUS_CLOSED = 4;
+enum ClearingEvent {
+  CLEARING_EVENT_UNSPECIFIED = 0;
+  CLEARING_EVENT_INSTANTIATED = 1;
+  CLEARING_EVENT_ACCEPTED = 2;
+  CLEARING_EVENT_FAILED = 3;
+  CLEARING_EVENT_INITIATOR_FUNDED = 4;
+  CLEARING_EVENT_RESPONDER_FUNDED = 5;
+  CLEARING_EVENT_POSTFUND_PROPOSED = 6;
+  CLEARING_EVENT_AGREED_ON_POSTFUND = 7;
+  CLEARING_EVENT_MOVE_TO_OPERATIONAL = 8;
+  CLEARING_EVENT_ISSUE_MARGIN_CALL = 9;
+  CLEARING_EVENT_PROCESS_MARGIN_CALL = 10;
+  CLEARING_EVENT_STARTED_SETTLEMENT = 11;
+  CLEARING_EVENT_ISSUE_POST_SETTLEMENT_MARGIN = 12;
+  CLEARING_EVENT_FINALIZE_SETTLEMENT = 13;
+  CLEARING_EVENT_FAILED_SETTLEMENT = 14;
+  CLEARING_EVENT_CHALLENGE = 15;
+  CLEARING_EVENT_FINALIZE = 16;
+  CLEARING_EVENT_WITHDRAW = 17;
+  CLEARING_EVENT_CLEAR_CHALLENGE = 18;
+  CLEARING_EVENT_CHALLENGE_TIMEOUT = 19;
```

Comment:
We have `TimeoutChallengeClearingEvent`. We should consider which variant will be better